src/components/formatters/test/formatter_json_rpc_test.cc

```diff
@@ -40,12 +47,27 @@
 
 namespace test {
 namespace components {
-namespace formatters {
+namespace formatters_test {
 
 using namespace NsSmartDeviceLink::NsSmartObjects;
 using namespace NsSmartDeviceLink::NsJSONHandler::Formatters;
 using namespace NsSmartDeviceLink::NsJSONHandler::strings;
 
+void CompactJson(std::string& str) {
```

Comment:
As far as I remember @pestOO, asked to caver this function with test too. I see no test.